shared/templates/kernel_build_config/oval.template

```diff
@@ -17,29 +17,34 @@
   </definition>
 
   <ind:textfilecontent54_test check="all"
-  comment="Check /boot/config-.* files for {{{ CONFIG }}}={{{ VALUE }}}"
+  comment="Check /boot/config-.* files for {{{ CONFIG }}}{{{ '=' + VALUE if VALUE else " according to " + VARIABLE }}}"
   id="test_kernel_{{{ CONFIG | lower }}}" version="1">
     <ind:object object_ref="object_kernel_{{{ CONFIG | lower }}}" />
+    <ind:state state_ref="state_kernel_{{{ CONFIG | lower }}}" />
   </ind:textfilecontent54_test>
 
   <ind:textfilecontent54_object id="object_kernel_{{{ CONFIG | lower }}}" version="1">
     <ind:filepath operation="pattern match">^/boot/config-.*$</ind:filepath>
-    <ind:pattern operation="pattern match">^{{{ CONFIG }}}="?{{{ VALUE }}}"?$</ind:pattern>
+    <ind:pattern operation="pattern match">^{{{ CONFIG }}}=(.*)$</ind:pattern>
```

Comment:
I believe this will break the double quotes matching
```suggestion
    <ind:pattern operation="pattern match">^{{{ CONFIG }}}="?(.*)"?$</ind:pattern>
```

Reply:
I've moved the double quote matching to https://github.com/ComplianceAsCode/content/pull/8492/files/da198a111e3ab8a168b091d6b9be3aabc543f139#diff-5e630ca5dad12e93ce7d1ed3511391ddb0e02ea9d95d9116b1c390746342d51cR38.
But I see now that the when the value comes from a string, we don't match the double quotes.